src/Components/Clock/Clock.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ConsoleClock } from '../ConsoleClock/ConsoleClock';
+import { Button } from '../Button/Button';
+import where from '../../image/4qb.gif';
+
+import './Clock.scss';
+
+export class Clock extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  }
+
+  showClock = () => {
+    this.setState({
+      isClockVisible: true,
+    });
+  }
+
+  hideClock = () => {
+    this.setState({
+      isClockVisible: false,
+    });
+  }
+
+  getRandomNameClock = () => {
+    this.setState({
+      clockName: Math.floor(Math.random() * 100),
+    });
+  }
+
+  render() {
+    const getClock = this.state.isClockVisible
```

Comment:
Звучить як функція, хоча ти просто вибираєш який компонент рендерити
```suggestion
    const сlockСomponent = this.state.isClockVisible
```